snapcraft/internal/elf.py

```diff
@@ -34,6 +36,11 @@
 logger = logging.getLogger(__name__)
 
 
+@functools.lru_cache()
+def _get_host_libc_path(arch_triplet: str) -> pathlib.Path:
+    return pathlib.Path("/lib") / arch_triplet / "libc.so.6"
```

Comment:
The only place where this is called, it does `str()` around the result.

So in reality, you could solve this with just an `os.path.join` without going back and forward with `pathlib`. 

Furthermore, probably this `os.path.join` could just happen where this function is called.

Reply:
I 100% agree, but we've been standardizing on using pathlib and converting to string as necessary. For that reason I plan to leave it as-is unless there's an issue.

Reply:
And now I'm using is_file() :D